dcos-log/api/v1/handlers.go

```diff
@@ -17,6 +17,15 @@ import (
 	"github.com/gorilla/mux"
 )
 
+type key int
+
+var streamKey key = 1
+
+func requestStreamKeyFromContext(ctx context.Context) bool {
```

Comment:
this code was there, it just lived in a different file but under the same package